addons/project/views/project_task_views.xml

```diff
@@ -562,7 +562,7 @@
                     <group>
                         <field name="project_id" invisible="1" />
                         <field name="company_id" invisible="1" />
-                        <field name="parent_id" domain="[('id', '!=', id)]" context="{'search_default_project_id': project_id, 'search_default_open_tasks': 1}" /> 
+                        <field name="parent_id" domain="[('id', '!=', id)]" context="{'search_default_project_id': project_id, 'search_default_open_tasks': 1, 'show_project_update': False}" />
```

Comment:
I think we should check the active_model to know if we have to display the project update button or not.

something like this:

```diff
diff --git a/addons/project/static/src/components/project_control_panel/project_control_panel.js b/addons/project/static/src/components/project_control_panel/project_control_panel.js
index bfa539f5b32c..56311a9405e4 100644
--- a/addons/project/static/src/components/project_control_panel/project_control_panel.js
+++ b/addons/project/static/src/components/project_control_panel/project_control_panel.js
@@ -10,8 +10,10 @@ export class ProjectControlPanel extends ControlPanel {
     setup() {
         super.setup();
         this.orm = useService("orm");
-        const { active_id, show_project_update } = this.env.searchModel.globalContext;
-        this.showProjectUpdate = this.env.config.viewType === "form" || show_project_update;
+        const { active_model, active_id, show_project_update } = this.env.searchModel.globalContext;
+        this.showProjectUpdate =
+            this.env.config.viewType === "form" ||
+            (show_project_update && active_model === "project.project");
         this.projectId = this.showProjectUpdate ? active_id : false;

         onWillStart(async () => {
```

Reply:
```patch
-            overflow: scroll hidden !important;
+            overflow: visible !important;
```
Breaks the snapping feature of the kanban on the horizontal axis (`scroll-snap-type: x` CSS rule).